packages/fuels-test-helpers/src/signers.rs

```diff
@@ -98,15 +99,16 @@ pub async fn launch_custom_provider_and_get_wallets(
 /// use fuels_test_helpers::setup_test_provider;
 ///
 /// async fn test_provider() -> Result<(), Box<dyn std::error::Error>> {
-///   let (_provider, _address) = setup_test_provider(vec![], None).await;
+///   let (_provider, _address) = setup_test_provider(vec![], None, None).await;
 ///   Ok(())
 /// }
 /// ```
 pub async fn setup_test_provider(
     coins: Vec<(UtxoId, Coin)>,
     node_config: Option<Config>,
+    messages: Option<Vec<Message>>,
```

Comment:
Should we move `messages` to be right after `coins`? And maybe either `coins` should be optional too, or we just have people specify `vec![]` if they don't want to provide either coins or messages?